scripts/ci/windows_flake_rate_proof.py

```
@@ -0,0 +1,305 @@
+# instrument is a different rate.
+INSTRUMENT_PATHS = (
+    "scripts/ci/windows_flake_rate_proof.py",
+    ".github/workflows/windows-flake-rate-proof.yml",
+)
+
+EXIT_INSTRUMENT_BROKEN = 2
+EXIT_COULD_NOT_MEASURE = 40
+
+
+def run(command: list[str], **kwargs) -> subprocess.CompletedProcess:
+    return subprocess.run(command, capture_output=True, text=True, check=False, **kwargs)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)